test/integration/minishift.go

```diff
@@ -23,10 +23,12 @@ import (
 	"fmt"
 	"github.com/minishift/minishift/test/integration/util"
 	"strings"
+	"sync"
 	"time"
 )
 
-var lastCommandOutput CommandOutput
+var commandOutputs []CommandOutput
```

Comment:
I actually considered moving `commandVariables` out, and rename as `scenarioVariables`...